src/v3/src/types/schema.ts

```diff
@@ -387,6 +394,7 @@ export interface ImageWithTextElement extends UISchemaElement {
     SVGIcon: FunctionComponent;
     textContent?: string;
     alignment?: string;
+    textClassName?: string;
```

Comment:
I'd like to avoid passing along className along if possible, since it will lead to us relying on className rather than other semantic attributes as a selector in the long term. Is it possible to use a more specific property to control the addition of `no-translate` as a class on each component? (e.g. `noTranslate?: boolean` with default `false`)

Reply:
Great idea actually, I disliked the approach I added but was hoping to get some feedback :D ty! I will update 🙏